test/functional/wallet_bumpfee.py

```diff
@@ -173,6 +177,29 @@ def test_small_output_fails(rbf_node, dest_address):
     rbfid = spend_one_input(rbf_node, dest_address)
     assert_raises_rpc_error(-4, "Change output is too small", rbf_node.bumpfee, rbfid, {"totalFee": 50001})
 
+def test_no_more_inputs_fails(rbf_node, dest_address):
+    # feerate rbf requires confirmed outputs when change output doesn't exist or is insufficient
+    rbf_node.generatetoaddress(1, dest_address)
+    # spend all funds, no change output
+    rbfid = rbf_node.sendtoaddress(rbf_node.getnewaddress(), rbf_node.getbalance(), "", "", True)
+    assert_raises_rpc_error(-4, "Unable to create transaction: Insufficient funds", rbf_node.bumpfee, rbfid)
+
+def test_small_output_with_feerate_succeeds(rbf_node, dest_address):
+    # Make sure additional inputs exist
+    rbf_node.generatetoaddress(101, rbf_node.getnewaddress())
+    rbfid = spend_one_input(rbf_node, dest_address, Decimal("0.00000700"))
+    assert_equal(len(rbf_node.getrawtransaction(rbfid, 1)["vin"]), 1)
+    # successfully bump this transaction a dozen times, even after tossing change
+    for i in range(12):
```

Comment:
Nit: `for _ in range(12):` is more idiomatic when `_` is unused :-)

Reply:
using it now :)